src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+:root {
+  --link-color: #00acdc;
+  --color: #616070;
+  --bg-color: #fff;
+  --test-color: #060b35;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__image img {
+  margin: 0 auto;
+  max-width: 160px;
+  display: block;
+  height: 100%;
+}
+
+.card__text {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--test-color);
```

Comment:
The custom property `--test-color` is used here. If you renamed it following the previous comment, make sure to update its usage here as well.